### DISCUSSION TITLE
Osiris cheat update

### DISCUSSION THREAD

**Discussion by SENAllu** [danielkrupinski/Osiris#3538] [General]
So, i hv now updated osiris and this update is for those who cant figure out to use VS 2019 or 2022.
https://anonfiles.com/Jf1az5Ffxc/Osiris_dll
its latest update is on feb 2 2022.

**Comment by JDM02152021** [comment]
preview??

**Comment by fanat1cc** [comment]
Your better of raring it and using Google drive since anon files link only last about 10-30days
